### PR TITLE
Add BAC calculator

### PR DESCRIPTION
**Pull request by swetasingh8844**
This is what the page looks like!!

image

### CODE REVIEW

**Comment by swetasingh8844** [comment]
@ritika728  please have a look

**Review by ritika728 — APPROVED** [review]

**Comment by ritika728** [comment]
@swetasingh8844 please resolve the conflicts

File: html_files/fitnesstracking.html
Comment (ritika728) [inline html_files/fitnesstracking.html:1]: The content for this calculator in the fitness tracking page can be improved by elaborating it. Kindly do this change

Comment (swetasingh8844) [reply html_files/fitnesstracking.html:1]: okk

**Comment by swetasingh8844** [comment]
@swetasingh8844  I changed it

**Comment by swetasingh8844** [comment]
@ritika728 I changed it

**Pull request merged**